apps/web/pages/booking/direct/[...link].tsx

```diff
@@ -52,50 +54,57 @@ const requestSchema = z.object({
           message: pageErrors.signature_mismatch,
           code: "custom",
         });
-        console.log(signedData, data, data[3], "==", sig);
       }
     }),
   reason: z.string().optional(),
 });
 
-function bookingContent(status: BookingStatus | undefined | null) {
+function bookingContent(status: BookingStatus | undefined | null, t: TFunction) {
   switch (status) {
     case BookingStatus.PENDING:
       // Trying to reject booking without reason
       return {
         iconColor: "gray",
         Icon: Icon.FiCalendar,
-        titleKey: "event_awaiting_approval",
```

Comment:
Why did you decide to convert this from a string to a function? Keeping it as a string makes this function a bit easier to implement, since it doesn't require the translations to be put in. Just wondering why, I can live with both solutions 😄